locale/de/LC_MESSAGES/django.po

```diff
@@ -2715,6 +2715,14 @@ msgid ""
 "\">further information here</a>.\n"
 "        "
 msgstr ""
+"\n"
+"Mit Ihrem Account auf Beteiligung.in können Sie sich an allen "
+#~ "öffentlichen Prozessen auf der Plattform beteiligen. Die Plattform bietet "
```

Comment:
should it be outcommented? Is that acomment in a po-file anyway?

Reply:
fixed